impl/src/main/java/org/ehcache/impl/internal/classes/ClassInstanceProvider.java

```diff
@@ -96,7 +97,7 @@ protected T newInstance(K alias, ServiceConfiguration<?> serviceConfiguration) {
     return newInstance(alias, config);
   }
 
-  private T newInstance(K alias, ClassInstanceConfiguration<T> config) {
+  private final T newInstance(K alias, ClassInstanceConfiguration<T> config) {
```

Comment:
No need for final, it's already private

Reply:
My bad... it became protected at one point and then moved back during my changes.